builder/oracle/oci/driver_oci.go

```diff
@@ -19,6 +23,30 @@ type driverOCI struct {
 	context       context.Context
 }
 
+var retryPolicy = &common.RetryPolicy{
+	MaximumNumberAttempts: 10,
+	ShouldRetryOperation: func(res common.OCIOperationResponse) bool {
+		var e common.ServiceError
+		if errors.As(res.Error, &e) {
+			if e.GetHTTPStatusCode() == 429 || e.GetHTTPStatusCode() == 500 || e.GetHTTPStatusCode() == 503 {
+				return true
+			}
```

Comment:
This  one could be a switch statement:
```suggestion
			switch e.GetHTTPStatusCode()  {
			case http.StatusTooManyRequests, http.StatusInternalServerError, http.StatusServiceUnavailable:
				return true
			}
```

(You'll need to import http first)

Reply:
It is indeed very pretty with the switch. I changed that now.